mbed_greentea/mbed_target_info.py

```diff
@@ -92,6 +92,20 @@
                 "reset_method": "default",
                 "program_cycle_s": 4
             }
+        },
+    "ARCH_BLE" : {
+        "yotta_targets": [
+                {
+                    "yotta_target": "tinyble-gcc",
```

Comment:
Yes!
See https://github.com/jacobrosenthal/target-tinyble-gcc/blob/master/target.json#L2 and https://github.com/jacobrosenthal/target-tinyble-gcc/blob/master/target.json#L21.